src/js/media-theme-element.js

```diff
@@ -49,12 +61,18 @@ export class MediaThemeElement extends window.HTMLElement {
   }
 
   get props() {
+    const observedAttributes = [
+      ...Array.from(this.attributes),
+      ...Array.from(this.mediaController?.attributes ?? [])
+        .filter(({ name }) => observedMediaAttributes[name])
+    ];
     const props = {};
-    for (let attr of this.attributes) {
+    for (let attr of observedAttributes) {
+      const name = observedMediaAttributes[attr.name] ?? camelCase(attr.name);
       if (attr.value != null) {
-        props[camelCase(attr.name)] = attr.value === '' ? true : attr.value;
+        props[name] = attr.value === '' ? true : attr.value;
       } else {
-        props[camelCase(attr.name)] = false;
+        props[name] = false;
       }
```

Comment:
^^ changes of this PR

the code below here is from the other PR https://github.com/muxinc/media-chrome/pull/383